README.md

```diff
@@ -1 +1,42 @@
-# Technical-documentation-page
\ No newline at end of file
+# Technical Documentation Page 🚀
+
+## Project Description 📝
+
+> This project is a documentation about PHP lunguage. by reading this documentation you can gain a general understanding of the PHP lunguage.
+I learned about how to create a responsive web page and how to write a code inside the page that show like a code.
+
+## Demo 📸
+
+Include a demo or animated GIF of your project. You can use tools like [asciinema](https://asciinema.org/) to record your terminal sessions.
+
+![Demo](images/capture)
+
+## Technologies Used 🛠️
+
+- HTML
+- CSS
+
+## Installation 💻
+
+To access this project, you need to follow the steps below.
+
+```bash
+git clone git@github.com:faridafaqiri/Technical-documentation-page.git
+
+```
+
+## Usage 🎯
+
+after clone the repository in your local machine you can open the html file in your browser.
+to access the code of project you can open it visual studio code editor.
+
+## Author 👩‍💻
+
+Provide your name and a link to your Social Media profiles. You can also include your email address.
```

Comment:
- Remove the instructions I added for you for each part.

Reply:
Hi @ahzamir  
thank you very much for taking the time to provide valuable feedback on my project.
Your insights were helpful and constructive, and I have made the necessary changes.